framework/graphics/vulkan_resources_util.cpp

```diff
@@ -1842,14 +1844,15 @@ VkResult VulkanResourcesUtil::ReadImageResources(const std::vector<ImageResource
                                    tmp_data[i].scaled_image,
                                    tmp_data[i].scaled_image_memory);
 
-                copy_image = tmp_data[i].scaled_image != VK_NULL_HANDLE ? tmp_data[i].scaled_image : copy_image;
-
                 if (result != VK_SUCCESS)
                 {
                     // free temporary resource, continue
                     tmp_data[i] = {};
                     continue;
                 }
+
```

Comment:
Two questions:
1) Why did you move it after the result check above?
2) Is this fixing anything from the old check or was this just a code readability change?